### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 43

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#231] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152255402-70eb132d-4d8e-475d-9952-9bc5d4e8a7e0.png)

**Comment by i0nut02** [comment]
1. (p + q -1  scelgo q-1)
2. (p - q + q -1 scelgo q-1) --> (p-1 scelgo q-1)
3. 30022001 ---> 3021 --->  4 * 3 * 2
